libcontainer/intelrdt/intelrdt.go

```diff
@@ -298,20 +312,26 @@ func isIntelRdtMounted() bool {
 	return true
 }
 
-func parseCpuInfoFile(path string) (bool, bool, error) {
-	isCatFlagSet := false
-	isMbaFlagSet := false
+type cpuInfoFlags struct {
+	isCatFlagSet      bool
```

Comment:
You can probably drop the word `Flag`, i.e. `isCatSet`, `isMbaSet` etc.

Come to think of it, you can also drop `is` and `Set`, too.